runtime.go

```diff
@@ -550,9 +570,33 @@ type ComponentScript struct {
 	Name string
 	// Function to render.
 	Function string
-	// Call of the function in JavaScript syntax, including parameters.
+	// Call of the function in JavaScript syntax, including parameters, and ensures parameters are HTML escaped.
 	// e.g. print({ x: 1 })
 	Call string
+	// Call of the function in JavaScript syntax, including parameters. Doesn't not HTML escape parameters; useful for directly calling.
```

Comment:
"Doesn't not HTML escape parameters"?

I think it should be "Does not HTML escape parameters.".

I think it's not clear why you'd want to use `Call` vs `CallInline` from the docstring. They're both a call in JavaScript, but one is intended for calling the component script within HTML event handler attributes, and the other is intended for calling the component script from anywhere.

The docstring example I added `{ x: 1}` doesn't make it clear that params would be HTML escaped in the `Call` string at all, which was probably OK before, but now there's a `CallInline`, it might make sense to clarify.

Reply:
The original doc string doesn't make too much sense on my first read, hence why the copy/paste. I'll update both to be clearer. I'm also not feeling super strongly on the `Call` versus `CallInline` naming and def open to better names.